File: Clase  - 10/src/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3 {

    public static void main(String[] args) {
        // Calcular la estación del año con switch

        Scanner entrada = new Scanner(System.in);
        System.out.println("Ingrese el mes del año: ");
        var mes = Integer.parseInt(entrada.nextLine());
        var estacion = "Estación desconocida";

        switch (mes) {
            case 1: case 2: case 3:
                estacion = "Verano";
                break;
            case 4: case 5: case 6:
                estacion = "Otoño";
                break;
            case 7: case 8: case 9:
                estacion = "Invierno";
                break;
            case 10: case 11: case 12:
                estacion = "Primavera";
                break;
            default:
                estacion = "Estación desconocida";
        }
        System.out.println("La estación para el mes " + mes + " es: " + estacion);
    }
}

File: Clase - 7/Ejercicios/Ejercicio3.py

mes_anio = int(input("Ingrese el mes del año entre 1 y 12: "))

if mes_anio <= 3:
    print("Es Verano")
elif mes_anio > 3 and mes_anio <= 6:
    print("Es Otonño")
elif mes_anio > 6 and mes_anio <= 9:
    print("Es Invierno")
elif mes_anio > 9 and mes_anio <= 12:
    print("Es primavera")
elif mes_anio > 1 or mes_anio < 12:
    print("El numero ingresado debe ser entre 1 y 12")